modules/mobile/app/services/mobile/v0/profile/sync_update_service.rb

```diff
@@ -58,7 +58,7 @@ def save!(http_method, resource_type, params)
         end
 
         def build_record(type, params)
-          if type == :address && Flipper.enabled?(:va_v3_contact_information_service, @user)
+          if type == :address && Flipper.enabled?(:mobile_v2_contact_info, @user)
```

Comment:
Mobile owns the Contact Information V2 migration for all mobile requests. Replaced va_v3_contact_information_service with mobile_v2_contact_info to prevent the newly updated Contact Information service to be used.